app/views/devise/sessions/new.html.slim

```diff
@@ -19,7 +16,7 @@ p.my3 == t('notices.log_in_consent.text', app: APP_NAME, link: link)
 .clearfix.pt1.border-top
   - if @sp_name && @sp_return_url
```

Comment:
I like that idea. I don't have a strong preference as to when we do this. Maybe a separate PR?